### ISSUE TITLE
rack-flash may have changed

### ISSUE THREAD

**Issue by micahshute** [learn-co-curriculum/playlister-sinatra#67]
Hi, I think rack flash may have changed - I wasn't able to get a rack-flash message to persist through a redirect - I was only able to get it to show if I rendered an erb file in the same block that the flash message was created.

**Comment by maxwellbenton** [comment]
Hi there @micahshute,

I'm having difficulty reproducing this issue. Using the solution branch, it is possible to persist a rack-flash message through a redirect when creating a song.

I am going to close this issue for now, but Iif you can provide further detail about the steps required to reproduce the issue and any additional relevant information about your development environment (local or IDE), I'd be happy to take a second look!

As always, thanks for contributing! 💙

**Comment by micahshute** [comment]
Update to this - so after trying the solution branch myself then going back to my project rack flash "magically" worked. Now, 2 months later, I was helping someone with this lab that had the same issue I did, I went back to my fixed project which was working, and now flash messages were no longer rendering. I re-ran the solution branch, then came back to my project and rack flash began working. The only difference I could find in the Gemfile.lock file was the bundler version, which was earlier for the solution branch.  Due to this information, the fact that the github page for rack flash recommends not using rack flash with Sinatra [at the top of the page](https://github.com/nakajima/rack-flash), and [other closed issues](https://github.com/learn-co-curriculum/playlister-sinatra/issues/60) with a similar theme (but closed because it is not reproducible on the solution branch), I recommend either investigating the cause of the errors (due to older versions of software in the solution branch), or switching the lab over to use Sinatra flash instead of rack flash.
